File: fibonacci/recursive.js
const utils = require('./utils');

const fibo = (n) => {
  if (n < 2) {
    return n;
  } else {
    return fibo(n-1) + fibo(n-2);
  }
}


const run = (n) => utils.bench(fibo)(n);


n = 37
s = utils.limit(`${run(n)}`, 10);
console.log(`n=${n} fib=${s}`);

File: fibonacci/recursive.py
import utils


def fibo(n):
    if n < 2:
        return n
    else:
        return fibo(n-1) + fibo(n-2)


@utils.bench
def run(n):
    return fibo(n)


n = 37
s = utils.limit("{}".format(run(n)), 10)
print('n={} fib={}'.format(n, s))
